demos/demos_by_use_case/bio/MetagenomicDemo.ipynb

```diff
@@ -0,0 +1,1086 @@
+      "source": [
+        "# install [HUMAnN 3](https://huttenhower.sph.harvard.edu/humann), a method for efficiently and accurately profiling the abundance of microbial metabolic pathways and other molecular functions from metagenomic or metatranscriptomic sequencing data."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "gtOG0QeoUoX6"
+      },
+      "source": [
+        "### takes very long for running all samples\n",
+        " (1day+ run on cluster)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "yIeeDXPBHN6D"
+      },
+      "outputs": [],
+      "source": [
+        "%%bash\n",
+        "\n",
+        "if [ ! -f PRJNA544527_mpa4out.txt ]; then\n",
+        "\n",
+        "    pip install humann --no-binary :all:\n",
+        "    pip install metaphlan\n",
+        "\n",
+        "    humann_databases --download utility_mapping full /path/to/databases --update-config yes\n",
+        "\n",
+        "    # humann_test\n",
+        "    wget https://github.com/biobakery/humann/raw/master/examples/demo.fastq.gz\n",
+        "    humann -i demo.fastq.gz -o sample_results\n",
+        "\n",
+        "\n",
+        "    mkdir assemble epi_sam_out mpa4_out\n",
+        "    humann -i /content/All_MAGs/Sample_101_S75_bin_1.fa -o test_out\n",
+        "\n",
+        "\n",
+        "    seq=$(ls /content/*.fastq | cut -d / -f2| cut -d _ -f1)\n",
+        "\n",
+        "    for i in $(eval \"echo \"$seq\" | cut -d _ -f1\")\n",
+        "\n",
+        "    do\n",
+        "    metaphlan /content/${i}.fa --nproc 40 --input_type fasta -o /content/assemble/${i}/h4_out.txt -t rel_ab_w_read_stats\n",
+        "    done\n",
+        "fi"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "dHbwBIEn6Wxr"
+      },
+      "source": [
+        "# umap and dbscan\n",
+        "\n",
+        "idea for metagenomic analysis based on [Quantifying Shared and Unique Gene Content across 17 Microbial Ecosystems\n",
+        "](https://journals.asm.org/doi/full/10.1128/msystems.00118-23)\n",
+        "\n",
+        "(analyze all samples run on cluster)\n",
+        "\n",
+        "also this [paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0997-x?ref=https://codemonkey.link#Sec7) and [method](https://github.com/marbl/Mash/blob/master/INSTALL.txt)\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "RoIBLY3-670T",
+        "outputId": "7c1d8f5f-fc9f-44d3-9a3a-a268edc4c00c"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "--2024-07-09 07:23:29--  https://raw.githubusercontent.com/dcolinmorgan/grph/main/PRJNA544527-meta_inf.txt\n",
```

Comment:
looks like still need to update this one

can now use: https://gist.github.com/lmeyerov/b650f1ef9e56c3f1888ebb009bc5ed46